test/preloaded_subinc/test_repo/src/build_defs/empty.build_defs

```diff
@@ -0,0 +1 @@
+foo()
```

Comment:
Should this be an empty file? If not then should there be an empty file test?

Reply:
maybe the name should be clearer - there is a comment in the BUILD_FILE above. This is a build_def which doesn't define anything but does use a build def defined elsewhere which we expect to be able to be used because the def has been preloaded